packages/react-sdk/src/components/BoardBar/ExportWhiteboardDialog.test.tsx

```diff
@@ -61,8 +76,10 @@ describe('<ExportWhiteboardDialog/>', () => {
       );
     };
 
-    jest.mocked(URL.createObjectURL).mockReturnValue('blob:url');
-    jest.mocked(createWhiteboardPdf).mockReturnValue(of(new Blob(['value'])));
+    vi.spyOn(URL, 'createObjectURL').mockReturnValue('blob:url');
```

Comment:
We already spy on it in `setupTests` ([ref](https://github.com/nordeck/matrix-neoboard/pull/554/files#diff-eadb1af897bc7fd7863b6a3c9d13e1e7630ce6191b5ef7c904d6052264b01a78R99)). To avoid confusion of spying on the spy e.g. when debugging this should use the existing spies instead.

Reply:
Can you pleas give an answer here?

Reply:
Oops it shouldnt have been I think. I will check that after the daily

Reply:
Ok this seems to work for all but 1 tests :D I will check why it fails even with an afterEach cleanup being called.

Reply:
ah nvm I typoed something. So it does work. Commit incoming.